ibm_concert/z_apars/inventories/host_vars/zos_host.yml

```diff
@@ -1,25 +1,20 @@
 ---
-################################################################################
-# Copyright (c) IBM Corporation 2025
-################################################################################
-
-################################################################################
-# Description of the properties used in this configuration:
-# - Property `PYZ` is the python installation home path on the z/OS managed node (target),
-#    e.g, pyz: "/usr/lpp/IBM/cyp/v3r8/pyz"
-# - Property `ZOAU` is the ZOAU installation home on the z/OS managed node (target),
-#    e.g, zoau: "/usr/lpp/IBM/zoautil"
-# - property `ansible_python_interpreter` is the z/OS managed node (target) Python
-#    binary installation path, e.g, ansible_python_interpreter: "{{PYZ}}/bin/python3"
-#
-# Note, PYZ and ZOAU environment variables must be configured.
-################################################################################
+ansible_host: # FQDN or IP here
+ansible_user: concert
+ansible_python_interpreter: "{{ PYZ }}/bin/python3"
 
-PYZ: "path_to_python"
-ZOAU: "path_to_zoau"
+# these will change depending on your system's installation of IBM Python and Z Open Automation Utilities (ZOAU)
+PYZ: "/shared/IBM/aln/v3r1/ent/pyz/pyz-311"
```

Comment:
shouldn't they be generic so the end users must update to match their environments?